### PR TITLE
feat: expose playground port

### PR DESCRIPTION
**Pull request by nachomazzara**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -56,6 +56,8 @@ EXPOSE 8000
 EXPOSE 8001
 # JSON-RPC port
 EXPOSE 8020
+# HTTP-PLAYGROUND port
```
Comment (leoyvens) [inline docker/Dockerfile:59]: A better description for this is `Indexing status port`

**Review by leoyvens — CHANGES_REQUESTED** [review]

**Review by leoyvens — APPROVED** [review]
Thanks!

**Comment by leoyvens** [comment]
@nachomazzara looks like this needs a rebase.

**Comment by nachomazzara** [comment]
@leoyvens  done

**Comment by nachomazzara** [comment]
any update on this @leoyvens ?

**Pull request merged**